docs/en/openmldb_quickstart.md

```diff
@@ -0,0 +1,379 @@
+
+#### 3.3.6 Exit the CLI
+
+```sql
+> quit;
+```
+
+Up to this point, you have completed all the development and deployment work based on the cluster version of OpenMLDB CLI, and have returned to the operating system command line.
+
+#### 3.3.7 Real-time feature extraction
+
+Notice: warning:: According to the default deployment configuration, the http port for apiserver deployment is 9080.
+
+Real-time online services can be provided through the following Web APIs:
+
+```
+http://127.0.0.1:9080/dbs/demo_db/deployments/demo_data_service
+        \___________/ \____/ \_____________/
+              | | |
+        APIServer address Database name Deployment name
+```
+
+The input data of the real-time request accepts the `json` format, and we put a line of data into the `input` field of the request. The following example:
+
+```bash
+curl http://127.0.0.1:9080/dbs/demo_db/deployments/demo_data_service -X POST -d'{"input": [["aaa", 11, 22, 1.2, 1.3, 1635247427000, "2021-05- 20"]]}'
+```
+
+The following is the expected return result for this query (the computed features are stored in the `data` field):
+
+```json
+{"code":0,"msg":"ok","data":{"data":[["aaa",11,22]]}}
+
+```
+
+#### 3.3.8 Result description of real-time feature extraction
+
+Real-time request (deployment execution) is SQL execution in request mode (request mode). Unlike batch mode (batch mode), request mode will only perform SQL extractions on request rows. In the previous example, the POST input is used as the request line, assuming this line of data exists in the table demo_table1, and execute SQL on it:
+```
+SELECT c1, c2, sum(c3) OVER w1 AS w1_c3_sum FROM demo_table1 WINDOW w1 AS (PARTITION BY demo_table1.c1 ORDER BY demo_table1.c6 ROWS BETWEEN 2 PRECEDING AND CURRENT ROW);
+```
+The specific extraction logic is as follows (the actual extraction will be optimized to reduce the amount of extraction):
+1. According to the request line and the `PARTITION BY` partition of the window, filter out the lines whose c1 is "aaa", and sort them according to c6 from small to large. So theoretically, the intermediate data table after partition sorting is shown in the following table. Among them, the first row after the request behavior is sorted.
+```
+ ----- ---- ---- ---------- ----------- --------------- - -------------
+  c1 c2 c3 c4 c5 c6 c7
+ ----- ---- ---- ---------- ----------- --------------- - -------------
+  aaa 11 22 1.2 1.3 1635247427000 2021-05-20
+  aaa 12 22 2.200000 12.300000 1636097890000 1970-01-01
+  aaa 11 22 1.200000 11.300000 1636097290000 1970-01-01
+ ----- ---- ---- ---------- ----------- --------------- - -------------
+```
+2. The window range is `2 PRECEDING AND CURRENT ROW`, so we cut out the real window in the above table, the request row is the smallest row, the previous 2 rows do not exist, but the window contains the current row, so the window has only Request line this line.
```

Comment:
Request line this line -> one row(the request row)

Reply:
thank you.